cli.js

```diff
@@ -57,18 +57,18 @@ var cli = meow([
 	'  ava [<file|directory|glob> ...]',
 	'',
 	'Options',
-	'  --init             Add AVA to your project',
-	'  --fail-fast        Stop after first test failure',
-	'  --serial, -s       Run tests serially',
-	'  --tap, -t          Generate TAP output',
-	'  --verbose, -v      Enable verbose output',
-	'  --no-cache         Disable the transpiler cache',
-	'  --no-power-assert  Disable Power Assert',
-	'  --match, -m        Only run tests with matching title (Can be repeated)',
-	'  --watch, -w        Re-run tests when tests and source files change',
-	'  --source, -S       Pattern to match source files so tests can be re-run (Can be repeated)',
-	'  --timeout, -T      Set global timeout',
-	'  --concurrency, -c  Maximum number of test files running at the same time (EXPERIMENTAL)',
+	'  --init				Add AVA to your project',
+	'  --fail-fast				Stop after first test failure',
+	'  --serial, -s			Run tests serially',
+	'  --tap, -t				Generate TAP output',
+	'  --verbose, -v			Enable verbose output',
+	'  --no-cache				Disable the transpiler cache',
+	'  --no-power-assert			Disable Power Assert',
+	'  --match, -m				Only run tests with matching title (Can be repeated)',
+	'  --watch, -w				Re-run tests when tests and source files change',
+	'  --source, -S			Pattern to match source files so tests can be re-run (Can be repeated)',
+	'  --timeout, -T			Set global timeout',
+	'  --concurrency, -c			Maximum number of test files running at the same time (EXPERIMENTAL)',
```

Comment:
Yes, there is a diff. But it lines up correctly in the terminal, when I execute it now. I'm not sure how to un-diff just that particular part of the file, although I know its possible through git.